bincapz segfaults if passed a bogus path name

If you run:

```she.ll
% bincapz /non/existent/file                                                                                                                                                                 
```

It crashes:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0x0 pc=0x102735f38]

goroutine 1 [running]:
github.com/chainguard-dev/bincapz/pkg/bincapz.RenderTable(0x103923678?, {0x103923698, 0x1400004a010})
	/Users/t/go/pkg/mod/github.com/chainguard-dev/bincapz@v0.1.0/pkg/bincapz/render.go:23 +0x28
main.main()
	/Users/t/go/pkg/mod/github.com/chainguard-dev/bincapz@v0.1.0/bincapz.go:71 +0x5ac
```

--- a/bincapz.go
+++ b/bincapz.go
@@ -51,6 +51,10 @@ func main() {
 	}
 
 	res, err := bincapz.Scan(bc)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "scan failed: %v\n", err)
+		os.Exit(3)
+	}
 
 	switch *formatFlag {
 	case "json":

--- a/pkg/bincapz/scan.go
+++ b/pkg/bincapz/scan.go
@@ -27,6 +27,7 @@ func findFilesRecursively(root string) ([]string, error) {
 	err := filepath.Walk(root,
 		func(path string, info os.FileInfo, err error) error {
 			if err != nil {
+				klog.Errorf("walk %s: %v", path, err)
 				return err
 			}
 			if info.IsDir() {

